sdk/include/reactos/roscompat.h

```diff
@@ -0,0 +1,32 @@
+/*!
+    \brief Header shared by ntdll/ldr and spec3def
+*/
+#pragma once
+
+#ifdef ROSCOMPAT_HOST
+#include <pecoff.h>
+#else
+#endif
+
+typedef enum _ROSCOMPAT_VERSION_BIT
```

Comment:
Why is this called "VERSION_BIT" if it is not a bitmask?

Reply:
it is the bit number of the bitmask.
so: 1<<0, 1<<1, 1<<2,...